app/index.js

```diff
@@ -73,12 +70,24 @@ app.log = new Winston.Logger({
 app.log.cli();
 
 /**
+ * Initialize database connection
+ */
+mongoose.connect(app.config.database.url);
+
+mongoose.connection.on('error', (msg) => {
+  app.log.error(msg);
```

Comment:
If we get an error Houston is not functional, so we should really fail here, right?